src/server/command_registry.cc

```diff
@@ -20,6 +20,7 @@ using namespace std;
 ABSL_FLAG(vector<string>, rename_command, {},
           "Change the name of commands, format is: <cmd1_name>=<cmd1_new_name>, "
           "<cmd2_name>=<cmd2_new_name>");
+ABSL_FLAG(vector<string>, restricted_commands, {}, "Commands restricted to admin users only");
```

Comment:
admin becomes overused in our codebase due to categories admin, and commands admin etc.
Maybe `Commands restricted to connections coming through admin port`

Reply:
sure updated